lib/rake/javaextensiontask.rb

```diff
@@ -108,8 +108,11 @@ def define_compile_tasks(for_platform = nil, ruby_ver = RUBY_VERSION)
         EOF
         warn_once(not_jruby_compile_msg) unless defined?(JRUBY_VERSION)
 
+        javac_path = File.exist?(t = File.join(ENV["JAVA_HOME"], "bin", "javac")) ?
```

Comment:
It seems that `ENV["JAVA_HOME"]` may not exist:

https://github.com/rake-compiler/rake-compiler/blob/150c8e359748a4e7582daea6ecbe7ecb063d3086/lib/rake/javaextensiontask.rb#L267-L268

Can we reuse the checked `ENV["JAVA_HOME"]` value?

Reply:
Not sure if `ENV["JAVA_HOME"]` is used anywhere?

Can we use:
```ruby
ENV.fetch("JAVA_HOME", "/")
```

Reply:
I just pushed a change to using `ENV.fetch`.  Used `"/"` for the default so it's an absolute path...

Reply:
Oh, sorry. It's `JRUBY_HOME` not `JAVE_HOME`...